app/helpers/check_answers_helper.rb

```diff
@@ -43,6 +43,6 @@ def total_applicable_questions(subsection, lettings_log, current_user)
   end
 
   def get_answer_label(question, lettings_log)
-    question.answer_label(lettings_log).presence || "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
+    question.answer_label(lettings_log, current_user).presence || "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
```

Comment:
Where is currently user coming from here?

Reply:
my guess is it's the devise controller method, which is also a helper method. this module is mixed in so is likely in scope

Reply:
Given this seems to be working fine and `current_user` was already being used in many places in this file, I think this is fine.